main.js

```diff
@@ -0,0 +1,47 @@
+// questions: 1) if there were multiple tables, how would createExItem know which table to append a new row to?
+// 2) is there a better way to combine mouse click and enter key to create a new row
+
+var exAmount = document.querySelector('input[name="expense-amount"]');
+var exName = document.querySelector('input[name="expense-name"]');
+var submitButton = document.querySelector('.add-expense');
+
+var createExItem = function() {
+
+  if (exAmount.value === "" && exName.value === ""){
+    alert("Enter Some Value");
+    } else {
+        var expenseAmount = exAmount.value; //gets expense amount
```

Comment:
This is when you have access to the expense amount, so where you should start thinking about tallying up expenses.